lib/Notification/Notifier.php

```diff
@@ -287,6 +298,75 @@ protected function shortenJsonEncodedMultibyteSave(string $subject, int $dataLen
 		return $temp;
 	}
 
+	private function parseStoredRecording(
+		INotification $notification,
+		Room $room,
+		Participant $participant,
+		IL10N $l
+	): INotification {
+		$shareAction = $notification->createAction()
+			->setParsedLabel($l->t('Share to chat'))
+			->setPrimary(true)
+			->setLink(
+				$this->urlGenerator->linkToRouteAbsolute(
+					'ocs.spreed.Chat.shareObjectToChat',
+					[
+						'apiVersion' => 'v1',
+						'token' => $room->getToken()
+					]
+				),
+				IAction::TYPE_POST
+			);
+		$dismissAction = $notification->createAction()
+			->setParsedLabel($l->t('Dismiss'))
```

Comment:
Not sure about the wording. Dismiss might be confused that it dismisses the recording file which is not the case. Maybe 
```suggestion
			->setParsedLabel($l->t('Dismiss notification'))
```

Reply:
Is the text a bit big now:

![image](https://user-images.githubusercontent.com/1079143/212998130-93aa5af7-e950-4db0-9fc9-c046b2ca3239.png)